rust/operator-binary/src/backend/mod.rs

```diff
@@ -45,6 +47,13 @@ pub struct SecretVolumeSelector {
     /// The name of the `Pod`'s `Namespace`, provided by Kubelet
     #[serde(rename = "csi.storage.k8s.io/pod.namespace")]
     pub namespace: String,
+
+    #[serde(
+        rename = "secrets.stackable.tech/kerberos.service.names",
+        default = "SecretVolumeSelector::default_kerberos_service_names",
+        deserialize_with = "SecretVolumeSelector::deserialize_str_vec"
+    )]
+    pub kerberos_service_names: Vec<String>,
```

Comment:
We need to map this to the `k8sSearch` backend somehow...

Reply:
On second thought, I think "you are responsible for setting up the correct principals" is fair for V1 at least.

Reply:
Basically: do we need to translate this into a label query if this is present on a `k8sSearch` volume, or do we just hope that the admin provisioned the correct principals?

Reply:
thx!